src/Workspaces/Core/Desktop/Workspace/Storage/PersistentStorageService.cs

```diff
@@ -112,7 +112,7 @@ private IPersistentStorage GetStorage(Solution solution, string workingFolderPat
                     // everything seems right, use what we have
                     if (storage?.WorkingFolderPath == workingFolderPath)
                     {
-                        storage.AddRefUnsafe();
+                        storage?.AddRefUnsafe();
```

Comment:
I believe the implication is this is unneeded: the if check could only practically succeed if workingFolderPath was non-null.